!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  BA_MAX_STEP: 10
  BLUR_SZ: 3
  BLUR_VAR: 1.0
  INIT_MAX_KP: 1
  KP_BLOCKS: 25
  LOOP_CLOSURE: false
  MAX_E_DIST: 1.0
  MAX_REPROJ_DIST: 1
  MIN_FEATURE_DIST: 0
  MONO_INLIER_THRES: 0.5
  MONO_MIN_DISP: 0.5
  PROP_POSE_ITER: 50
  PROP_POSE_PYMD: 1
  QUAL_LEVEL: 0.1
  SCALE_MAX_STEP: 50
  SCALE_PYMD: 2
  TIME_DEBUG: false
  TIME_LOG: false
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      BA_MAX_STEP: 10
      BLUR_SZ: 3
      BLUR_VAR: 1.0
      INIT_MAX_KP: 1
      KP_BLOCKS: 25
      LOOP_CLOSURE: false
      MAX_E_DIST: 1.0
      MAX_REPROJ_DIST: 1
      MIN_FEATURE_DIST: 0
      MONO_INLIER_THRES: 0.5
      MONO_MIN_DISP: 0.5
      PROP_POSE_ITER: 50
      PROP_POSE_PYMD: 1
      QUAL_LEVEL: 0.1
      SCALE_MAX_STEP: 50
      SCALE_PYMD: 2
      TIME_DEBUG: false
      TIME_LOG: false
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
state: []
